backend/app/services/chatbot_service.py

```
@@ -18,6 +18,13 @@
 import shutil
 import os
 from processing.kb import build_knowledge_base, rebuild_kb_after_upload
```

Ruff (0.8.2)

`processing.kb.rebuild_kb_after_upload` imported but unused

Remove unused import: `processing.kb.rebuild_kb_after_upload`

(F401)